install_config/install/docker_registry.adoc

```diff
@@ -169,9 +169,41 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
-[[viewing-logs]]
 
+[[registry-ip-address]]
+=== Registry IP Address
+
+Because the integrated registry is referenced by its service IP address,
+if you decide to delete and recreate the docker-registry service,
+you should either take care to re-use its address,
+or reboot the cluster after creating the new docker-registry service.
+
+[[re-using-the-address]]
+==== Re-using the Address
+
+To re-use the IP address, you must save the IP address of the old docker-registry
+service prior to deleting it, and arrange to replace the newly assigned IP address
+with the saved one in the new docker-registry service.
+
+// NB: Snarfed from <https://github.com/openshift/openshift-docs/issues/1494>.
+. Make a note of the `ClusterIP` for the service.
+. `oc delete svc/docker-registry dc/docker-registry`
+. `oadm registry ... -o yaml > registry.yaml`
+. Edit *_registry.yaml_*, find the `Service` there,
+and set its `ClusterIP` to the address noted in step 1.
+. `oc create -f registry.yaml`
```

Comment:
The above doesn't seem to mark up correctly. Too many periods before each command?